### PR TITLE
A few SG card fixes

### LINKED ISSUE #5687: Tread Lightly increases rez costs of upgrades
Used Tread Lightly on a remote server and it also increased the rez costs of the upgrades installed in the server.

### LINKED ISSUE #5692: Anoetic Void ends the run, but still triggered successful
Spraybizzle3 approaches ICE protecting Server 2 at position 0.
EricF pays 0  to rez Anoetic Void.
Spraybizzle3 passes ICE protecting Server 2 at position 0.
Spraybizzle3 approaches Server 2.
EricF has no further action.
EricF pays 2  and trashes 2 cards (a card in HQ, a card in HQ) from HQ to use Anoetic Void to end the run.
Spraybizzle3 uses Dirty Laundry to gain 5 .
Spraybizzle3 is ending their turn 14 with 14  and 3 cards in their Grip.

### PR DESCRIPTION
**Pull request by merrickbrown**
fixes #5692 fixes #5687

### CODE REVIEW

**Comment by NoahTheDuke** [comment]
#5676 took care of the Anoetic Void bug.

**Comment by merrickbrown** [comment]
oops sorry about that

File: src/clj/game/cards/events.clj [hunk 1]
```diff
@@ -3020,7 +3020,7 @@
                       card
                       {:type :rez-additional-cost
                        :duration :end-of-run
-                       :req (req true)
+                       :req (req (ice? target))
```
Comment (NoahTheDuke) [inline src/clj/game/cards/events.clj:3023]: i'd love to get a PR to fix this!

Comment (merrickbrown) [reply src/clj/game/cards/events.clj:3023]: new PR #5698

**Pull request closed without merging**